src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/ChangePolicy.kt

```diff
@@ -53,14 +52,14 @@ data class ChangePolicy(
                 xcp.nextToken()
 
                 when (fieldName) {
-                    POLICY_NAME_FIELD -> policyName = xcp.text()
-                    STATE_FIELD -> state = xcp.textOrNull()
+                    ManagedIndexConfig.POLICY_ID_FIELD -> policyID = xcp.text()
+                    StateMetaData.STATE -> state = xcp.textOrNull()
```

Comment:
np: Let's add back this classes own POLICY_ID_FIELD and STATE_FIELD, I know they are technically the same, but if we for whatever reason decided to rename StateMetaData.STATE to "something_else" it shouldn't impact ChangePolicy since this one would still be "state".

Reply:
Shouldn't the `ChangePolicy` follow what ever the change we make to state if it is a different name? `ChangePolicy` should be combining existing model names and add it as part of its own model since the nature of this class is to indicate what changes will happen to other models.

Reply:
There is a resource mapping the need to be mapped with certain string. So it make sense `ChangePolicy` to have its own string const that is the same as the Index mapping. I will make the changed necessary